deps/rabbitmq_prometheus/README.md

```diff
@@ -66,7 +66,14 @@ When metrics are returned per object, nodes with 80k queues have been measured t
 In order to not put unnecessary pressure on your metrics system, metrics are aggregated by default.
 
 When debugging, it may be useful to return metrics per object (unaggregated).
-This can be enabled on-the-fly, without restarting or configuring RabbitMQ, using the following command:
+
+This can be done by scraping the `/metrics/per_object` endpoint:
```

Comment:
Can we please change this to `/metrics/per-object`? Our existing HTTP API endpoints uses dashes and not underscores. An atom with a dash would need quoting: `'per-object'`.

Reply:
That sounds reasonable. We originally went with `per_object` because that was consistent with the setting in the config, presumably because of the need for quoting atoms with dashes.